plugin/vimux.vim

```diff
@@ -197,11 +197,7 @@ function! VimuxTmux(arguments) abort
     echom l:tmuxCommand
   endif
   if has_key(environ(), 'TMUX')
-    let l:output = system(l:tmuxCommand)
-    if v:shell_error
-      throw 'Tmux command failed with message:' . l:output
-    endif
-    return l:output
+    return system(l:tmuxCommand)
```

Comment:
This should be restored, otherwise this looks good!

Reply:
Ups! Forgot to revert that one :facepalm: 
I've now removed that commit from the PR!